conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -52,6 +52,10 @@ class SysmanDeviceTest : public lzt::SysmanCtsClass {};
 #define SYSMAN_DEVICE_TEST SysmanDeviceTest
 #endif // USE_ZESINIT
 
+#ifdef USE_ZESINIT
+bool is_uuid_pair_equal(uint8_t *uuid1, uint8_t *uuid2);
```

Comment:
static ?

Reply:
As the function is within an unnamed/anonymous namespace, which provides internal linkage / encapsulation / other capabilities, it appears that static is not required. Please confirm. Thank you.

Reply:
Got it.. Didnt see the anonymous namespace there